migrations/educandu-2023-02-17-02-add-accredited-editors-to-documents.js

```diff
@@ -0,0 +1,23 @@
+export default class Educandu_2023_02_17_02_add_accredited_editors_to_documents {
```

Comment:
To clarify, there is a
`educandu-2023-02-17-01-add-protected-flag-to-documents.js`
migration before this and a 
`educandu-2023-02-20-01-repurpose-homepage-info-field-in-settings.js`
migration after this, on the main branch.

So this can stay named as is, it won't have a bad impact. Up to you.

Reply:
Perfect, I'll leave it then, it doesn't interfere with the latter one.